src/shared/components/data-table/DataTable.scss

```diff
@@ -0,0 +1,54 @@
+@import 'src/styles/common';
+
+.wrapper {
+  --table-background: $white;
+  overflow: auto;
+
+  &ThemeDark {
+    background-color: $light-grey;
+  }
+}
+
+.table {
+  width: var(--table-width, 100%);
+  height: var(--table-height, 500px);
+  overflow: auto;
+  display: block;
+
+  --table-background: $white;
```

Comment:
- Why are we defining the table-background variable multiple times here?
- Why do you need to define `--table-head-background` separately? Isn't the Table component already doing this?

Reply:
I've removed `table-background` defined inside the wrapper and `--table-head-background` entirely from this file.